src/components/ProductPage/ProductRipcord.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import numeral from 'numeral'
+import { InputProps } from 'react-neu'
+
+import SimpleButton from 'components/SimpleButton'
+import { ProductPageSection } from './ProductPageLayouts'
+import useWallet from 'hooks/useWallet'
+import { useRipcord } from 'hooks/useRipcord'
+import { TokenDataProps } from './ProductDataUI'
+import { Ethereum2xFlexibleLeverageIndex } from 'constants/productTokens'
+
+interface ProductRipcordProps extends InputProps {
+  tokenData: TokenDataProps
+}
+
+const ProductRipcord: React.FC<ProductRipcordProps> = ({ tokenData }) => {
+  const { isMetamaskConnected } = useWallet()
+  const handleRipcord = useRipcord()
+
+  const formatLeverageMetric = (metricValue: number) =>
+    numeral(metricValue).format('0.00').toString().toUpperCase()
+
+  const realLeverage =
+    tokenData.components && tokenData.components.length > 0
+      ? Number(tokenData.components[0].percentOfSet || 0) / 100
+      : 2
+
+  const ETH2xMaxLeverageRatio = 2.7
+  const BTC2xMaxLeverageRatio = 2.4
+
+  const maxLeverage = Ethereum2xFlexibleLeverageIndex.symbol
+    ? ETH2xMaxLeverageRatio
+    : BTC2xMaxLeverageRatio
+
+  const renderRealLeverage = (leverage: number) => {
+    if (leverage < maxLeverage) {
+      return (
+        <StyledPositiveChange>
+          {formatLeverageMetric(leverage)}x
+        </StyledPositiveChange>
+      )
+    }
+    return (
+      <StyledNegativeChange>
+        {formatLeverageMetric(leverage)}x
+      </StyledNegativeChange>
+    )
+  }
+
+  return (
```

Comment:
Can we make the styling in this section consistent with the `Market Data` section?
![image](https://user-images.githubusercontent.com/7955024/130649510-e85fb365-12dc-4398-b231-31bdf5014aff.png)